src/app/participant/forms.py

```diff
@@ -0,0 +1,6 @@
+__author__ = 'alex'
+from wtforms import Form, StringField, SelectField, TextAreaField
+
+
+class ParticipantsForm(Form):
+    name = StringField('Name')
```

Comment:
how do you set its league? this is kind of useless without being able to pick which league it belongs in. Creating participants through api would be easier for testing than this.